### PR TITLE
Add fix to acceptance tests for proxy support

### PR DESCRIPTION
**Pull request by Vasundharashukla**
Signed-off-by: Vasundhara Shukla <vasundharas@vmware.com>

1. **What this PR does / why we need it**:
        Fixing acceptance tests for proxy support.

### CODE REVIEW

**Review by ishangupta-ds — APPROVED** [review]

File: internal/authctx/helper.go [hunk 1]
```diff
@@ -91,10 +91,10 @@ func ProviderConfigureContext(_ context.Context, d *schema.ResourceData) (interf
 	config.TLSConfig.ClientAuthKey, _ = d.Get(clientAuthKey).(string)
 	config.TLSConfig.CaCert, _ = d.Get(caCert).(string)
 
-	return setContext(config)
+	return setContext(&config)
 }
 
-func setContext(config TanzuContext) (TanzuContext, diag.Diagnostics) {
+func setContext(config *TanzuContext) (TanzuContext, diag.Diagnostics) {
```
Comment (shreyassreenivas) [inline internal/authctx/helper.go:97]: @ishangupta-ds , the proxy changes was manually tested right ?
why this change now?

Comment (ishangupta-ds) [reply internal/authctx/helper.go:97]: Yeah, the original one should also work as we are just initialising the field here but since for all other functions we have passed as pointer so for consistency we changed here as well.

**Review by shreyassreenivas — APPROVED** [review]
